### PR TITLE
Add PostgreSQL ODBC driver

### PR DESCRIPTION
**Pull request by gongled**
Hi, @andyone.

I found that `postgresql-odbc` package provides pretty old version of ODBC driver, so we unable to communicate with newer versions of PostgreSQL (and PGBouncer) via Clickhouse ODBC bridge through unixODBC. I ask you to build and publish RPM package with updated version (ex. `10.03.0000`).

### CODE REVIEW

File: specs/postgresql-odbc/postgresql-odbc.spec [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+# actually come from the Postgres source tree, but we haven't got that
+# available while building this RPM.)  To generate: in psqlodbc source tree,
+#   aclocal -I . -I $PGSRC/config
+# then strip aclocal.m4 down to just the PGAC macros.
+# BUT: as of 09.01.0200, configure.ac hasn't been updated to use latest
+# PG macros, so keep using the previous version of acinclude.m4.
+Source1:              acinclude.m4
+
+BuildRequires:        gcc unixODBC-devel
+BuildRequires:        libtool automake autoconf postgresql10-devel
```
Comment (andyone) [inline specs/postgresql-odbc/postgresql-odbc.spec]: I'm not sure that `make` is a dependency for `automake` or not. Maybe we should specify `make` explicitly?

File: specs/postgresql-odbc/postgresql-odbc.spec [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+# CAUTION: acinclude.m4 has to be kept in sync with package's aclocal.m4.
+# This is a kluge that ought to go away, but upstream currently isn't
+# shipping their custom macros anywhere except in aclocal.m4.  (The macros
+# actually come from the Postgres source tree, but we haven't got that
+# available while building this RPM.)  To generate: in psqlodbc source tree,
+#   aclocal -I . -I $PGSRC/config
+# then strip aclocal.m4 down to just the PGAC macros.
+# BUT: as of 09.01.0200, configure.ac hasn't been updated to use latest
+# PG macros, so keep using the previous version of acinclude.m4.
+Source1:              acinclude.m4
```
Comment (andyone) [inline specs/postgresql-odbc/postgresql-odbc.spec:59]: It's not ok to define sources in different parts of the spec file. Could you please move it to Source0?

Comment (gongled) [reply specs/postgresql-odbc/postgresql-odbc.spec:59]: Oops, my bad. Fixed (dc9951d).

File: specs/postgresql-odbc/postgresql-odbc.spec [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+BuildRequires:        gcc unixODBC-devel
+BuildRequires:        libtool automake autoconf postgresql10-devel
+BuildRequires:        openssl-devel krb5-devel pam-devel zlib-devel readline-devel
```
Comment (andyone) [inline specs/postgresql-odbc/postgresql-odbc.spec:67]: Auto requirements search is a good feature, but I think we should specify all non-devel packages as dependencies.

File: specs/postgresql-odbc/postgresql-odbc.spec (hunk 1 above)
Comment (gongled) [reply specs/postgresql-odbc/postgresql-odbc.spec]: Done (3231c0a).

**Comment by andyone** [comment]
If this PR contained `bibop` tests, I would be so happy. 😉

File: specs/postgresql-odbc/postgresql-odbc.spec (hunk 3 above)
Comment (gongled) [reply specs/postgresql-odbc/postgresql-odbc.spec:67]: Fixed (`814bc9d`).

**Comment by gongled** [comment]
> If this PR contained `bibop` tests, I would be so happy. 😉

I have tried to write my first bibop recipe (f783386). How does it look?

**Pull request merged**